resources/groundwork/web.md

```diff
@@ -49,7 +49,64 @@ Have some technologies you enjoy using? See our [Contribution Guide](/CONTRIBUTI
 
 ### MERN Stack
 
-TODO
+| Pros | Cons |
+|------|------|
+| You can use either JavaScript or TypeScript with React and Node.js. This can minimize the cognitive overhead of switching between stacks, allowing your team to be more productive. | React has a steeper learning curve than other front-end libraries/frameworks. |
```

Comment:
This table should provide a relatively high-level overview of this stack's advantages, so I think we can word this a bit more succinctly as:

```suggestion
| The frontend and backend share similar languages and libraries. | React has a steeper learning curve than other front-end libraries/frameworks. |
```